lib/modules/commentQuickCollapse.js

```diff
@@ -35,3 +102,27 @@ function hideCommentsOnHeaderDoubleClick() {
 		}
 	});
 }
+
+function toggleCommentsOnClickLeftEdge(container) {
+	container = container || document.body;
+	const stylesheet = toggleCommentsOnClickLeftEdgeStylesheetTemplate({
+		bColor: module.options.leftEdgeColor.value,
+		hColor: module.options.leftEdgeHoverColor.value,
+		cColor: module.options.leftEdgeCollapsedColor.value,
+		widthI: 2 * (parseInt(module.options.leftEdgeWidth.value, 10) / 100),
+		widthO: 2.5 * (parseInt(module.options.leftEdgeWidth.value, 10) / 100),
+	});
+
+	addCSS(stylesheet);
+}
+
+function scrollOnCollapse() {
+	$(document.body).on('click', '.expand', function(event) {
```

Comment:
Flow wants this disambiguated with `event: Event` (as opposed to the `MouseEvent` specialization for `click`)